lab_1_keywords_tfidf/main.py

```diff
@@ -17,7 +18,15 @@ def clean_and_tokenize(text: str) -> Optional[list[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(text, str):
+        return None
+    text = text.replace('\n', ' ').lower().strip()
+    fin_text = ''
+    for symbol in text:
+        if symbol.isalnum() is True or symbol == ' ':
```

Comment:
simplify the condition. Don't use `== True`

Reply:
fixed